### ISSUE TITLE
Installation ISsues

### ISSUE THREAD

**Issue by huvec** [erfajo/OrchidForDynamo#44]
Revit: 2019.2.2
Dynamo: 2.03
Orchid: 203.1.1.7205

Hi There, 
I am not able to install Orchid succesfully. The Steps I have down are.

1. Install from Package Manager. I was directed to the GIthub page.

2. I downloaded the packges from Github and replaced all the files in the Orchid folder with the ones from Orchid_200 Folder . After doing this,  the Orchid is no longer in the dynamo interface. The Arrow show where it was before it disappeared.

Any help appreciated.
   
![image](https://user-images.githubusercontent.com/16100298/66981104-4bcc6280-f10f-11e9-8f71-65be9ae245c6.png)

**Comment by erfajo** [comment]
I have an idea...

how does your folder one level deeper look like?

I publish two flavors of every version, one for usage together with Revit and one for the “Sandbox version” which does not open from inside Revit. You need to take the flavor you need. I will presume this would be the “Dynamo Revit” flavor. Content in this folder is the one that should be placed in the “Orchid” folder. If you used the executable installer this would have been set up for you.
![image](https://user-images.githubusercontent.com/2844374/66982069-b5b81e00-f0b5-11e9-84c1-fd6a59975227.png)

**Comment by huvec** [comment]
This is what it look like at the moment.
![image](https://user-images.githubusercontent.com/16100298/66982747-54269c80-f113-11e9-9fe8-8dc4eb096776.png)

I copied the ones from dynamo revit folder.
I did try the installer but it did not work for me.

**Comment by erfajo** [comment]
Try to remove Lunchbox and try opening once more....

**Comment by huvec** [comment]
I have removed lunchbox but orchid has not returned.

**Comment by erfajo** [comment]
then move Lunchbox back...
could you try to remove all other packages than mine from the folder and then try to reopen... some package must contain a dynamo assembly and that is not allowed. Since I dont know what other packages do I cant say which, but I know that an old version of Lunchbox had this problem...

**Comment by huvec** [comment]
Here is it.
your package does not appear in the list. However the package is still considered to be installed.

![image](https://user-images.githubusercontent.com/16100298/67133051-9bbe3d00-f267-11e9-8aeb-40d68bf98bea.png)

**Comment by erfajo** [comment]
I have no idea why it doesn't load, so I am afraid I cant help you.

**Comment by erfajo** [comment]
I have uploaded a new build.
However, i did test downloading the repository and tried different ways to take files from the zip file. If i didn't unzip correctly, then the files failed over and over again. While unzipping correctly, then the files worked. They might have worked all along if I had done it correctly from the beginning.

**Comment by huvec** [comment]
Unfortunate to say it is still not working for me. 
Things I have tried.
1. Repair Dynamo
2. Download Orchid from package manager again.
3.Dowload Orchid from GitHub... Unzip the files and copy content into folders. 

I was reading issue #45 and also want to say our dynamo package are deployed from a central server. as well. However, I am installing orchid just for myself. 
I am going to assume that it may be something on my end that is causing the issue. I will chip away  at trying to make it work or pick up python to do what I need. I will keep an eye out on #45 and see if they have better success.

**Comment by erfajo** [comment]
I did the same, downloaded from Github, unzipped the file and copied the content. If I just opened the zip file in explorer and dragged the needed subfolder, then did it fail.
However, did you try to see if some of this information could affect you? https://github.com/DynamoDS/Dynamo/wiki/How-to-Utilize-Dynamo-Builds
Downloading my package and unzipping manually could give the same errors as if you downloaded beta builds of dynamo…
